doc/api/cli.md

```diff
@@ -969,25 +969,24 @@ In case an option value happens to contain a space (for example a path listed in
 ```
 
 Node.js options that are allowed are:
-- `--report-directory`
-- `--report-filename`
-- `--report-on-fatalerror`
-- `--report-on-signal`
-- `--report-signal`
-- `--report-uncaught-exception`
 - `--enable-fips`
+- `--es-module-specifier-resolution`
 - `--experimental-modules`
+- `--experimental-policy`
 - `--experimental-repl-await`
 - `--experimental-report`
 - `--experimental-vm-modules`
 - `--experimental-wasm-modules`
+- `--experimental-worker`
```

Comment:
This option is actually a no-op. Should this instead be added to the list of intentionally undocumented options in the test?

Reply:
I've moved it across as an intentionally undocumented option.